doc/source/usersguide/installation.rst

```diff
@@ -153,6 +153,25 @@ worry too much ;)
 
           $ C_INCLUDE_PATH=/usr/lib/openmpi/include pip install --upgrade tables
 
+    Since PyTables 3.2 can also query the *pkg-config* database to find the
```

Comment:
Maybe this rephrasing would be better? "Starting from PyTables 3.2, setup.py can also query..."